html/php-pre-challenge2/index.php

```diff
@@ -1,11 +1,17 @@
 <?php
 $array = explode(',', $_GET['array']);
 
-// 修正はここから
-for ($i = 0; $i < count($array); $i++) {
-
+$count = count($array);
```

Comment:
（修正不要の指摘です）

今回はミニコードなのでこれでOKです。
一般論としては、
プログラムのサイズが大きくなってくると、$count(数、という意味）だけでは後から（※1）見返してわかりにくい表記になってしまうので避けたほうがいいでしょう。

実際には下記のように対応することが多いです。
- 今回のように10〜20行（1つの画面で見れるくらいなら$countのままでOK）
- それ以上にならなら、もう少し親切な名前にする（$input_array_sizeとか、$max_countとか、etc）
   - とはいえ、処理を分解して10〜20行ごとにできないか検討する（関数を使うということですね）


※1
- プログラムの下の方で見かける
- 3ヶ月後にコードを修正する

```suggestion
$count = count($array);
```

Reply:
ありがとうございます！気をつけます！